charts/ocis/values.yaml

```diff
@@ -108,11 +110,14 @@ registry:
   # -- Configure the service registry type.
   # Defaults to "kubernetes".
   # Can be set to "etcd" or "natsjs", then the address of etcd / nats node(s) needs to be set to `registry.nodes`.
-  type: kubernetes
+  type: nats-js-kv
   # -- Nodes of the service registry to use.
-  nodes: []
-  # nodes:
-  #   - etcd.ocis-etcd.svc.cluster.local:2379
+  nodes:
+    - nats.ocis-nats.svc.cluster.local:4222
```

Comment:
Probably we should restructure this like for the message bus with the `external` part.

The logic behind it is that we have a "secret" / immutable default configuration and a mode where you gain full control

Reply:
or we do it like for the store where the nodes are set to `- "{{ .appNameNats }}:9233"`

Reply:
Your current solution has the major disadvantage that it only works if you deploy it to `ocis-nats` namespace